python/xorbits/pandas/__init__.py

```diff
@@ -26,6 +27,8 @@
 from . import accessors, core, groupby, plotting, window
 from .core import DataFrame, Index, Series
 
+PANDAS_MODULE_METHODS: Optional[Dict[str, Callable]] = None
```

Comment:
Maybe we can implement a function called `get_pandas_module_methods` with LRU cache size 1? and we can avoid global variables.

Reply:
And maybe we can put the function into `pandas_adapters` to keep consistency with `collect_pandas_moldule_members`?

Reply:
Oh, it's actually coming from the pandas_adapters, so refer to the next comment.